### PR TITLE
zmarkdown: Add statistics facility

### PR DESCRIPTION
**Pull request by artragis**
# goal of this PR

On ZDS we have an estimated time of reading for all contents.

For now this indication is not reliable at all because it uses a bare `len(text)` to evaluate the size of content.
This is problematic because it includes mardown-syntax and URL related chars.

In this PR I evaluate the true number of chars by visiting text nodes.

### CODE REVIEW

File: packages/zmarkdown/README.md [hunk 1]
```diff
@@ -73,6 +77,12 @@ Only `metadata` is described in the **Response** sections below.
 
   A list of unique languages used in GitHub Flavoured Markdown fences with a flag.
 
+* `metadata.stats` , object
+
+  stats about the parsed text:
+   - `signs`: number of chars, spaces uncluded
```
Comment (vhf) [inline packages/zmarkdown/README.md]:
```suggestion
   - `signs`: number of chars, spaces included
```

**Comment by artragis** [comment]
You've got a point

I will change it asap.

**Comment by vhf** [comment]
> I think `signs += node.value.length + 1` wouldn't help because `a *b* c` would be 8 instead of 5, right?

As expect, 45307bc gives `8` chars for `a *b* c`. I think that's too far off.

> What about adding something like
> 
> ```js
>       visit(tree, 'paragraph', (node) => {
>         signs += 1
>       })
> ```
> 
> before or after visiting `text`?

Your thoughts on this?

**Comment by artragis** [comment]
I just messed up.

**Comment by artragis** [comment]
should be good now

File: packages/zmarkdown/README.md (hunk 1 above)
Comment (vhf) [reply packages/zmarkdown/README.md]: @artragis

**Comment by vhf** [comment]
🚀 https://www.npmjs.com/package/zmarkdown/v/5.15.0

**Pull request merged**